.github/workflows/release-python.yml

```diff
@@ -5,17 +5,27 @@ on:
     types: [published]
   workflow_dispatch:
     description: "Manually publish release"
+    inputs:
+        publishWheels:
+          description: "Build and publish wheels to PyPI"
+          type: boolean
+          default: false
 
 jobs:
   is-python-release:
-    if: ${{ github.event_name == 'workflow_dispatch' || startsWith(github.event.release.tag_name, 'python/v') }}
+    if: github.event_name == 'workflow_dispatch' || startsWith(github.event.release.tag_name, 'python/v')
     runs-on: ubuntu-latest
     steps:
       - run: echo "Release tag starts with python/v, proceeding with release"
+  should-publish-wheels:
+    if: (github.event_name == 'workflow_dispatch' && inputs.publishWheels) || (github.event_name == 'release' && !github.event.release.prerelease)
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Publishing wheels (${{ github.event_name == 'release' }} && ${{ !github.event.release.prerelease }}) ${{ (github.event_name == 'workflow_dispatch' && inputs.publishWheels) || (github.event_name == 'release' && !github.event.release.prerelease) }})"
```

Comment:
Is including the condition in the `echo` necessary? I would expect that the `echo` only happens if the condition is true.

Reply:
Oops, it definitely is not. That was me trying to debug the condition. I was having an issue where the condition was `false`, but the job was still running. Printing it helped me learn the condition was indeed false, and the formatting in the condition was a problem. Removed it.